mon/entrypoint.sh

```diff
@@ -17,6 +17,17 @@ if [ ! -n "$MON_IP" ]; then
    exit 1
 fi
 
+# When using contanier network stack we use the first available ip
+if [ "$MON_IP" == "container" ]; then
+   MON_IP=$(hostname --ip-address)
```

Comment:
It looks like inside a container it returns the ip you have in /etc/hosts for the container, the way the container is build, it returns only one ip.
In other systems , yes it can return multiple ips